tests/_regtest_outputs/test_run_topostats.test_process_scan_lower.out

```diff
@@ -1,3 +1,3 @@
                    centre_x    centre_y  radius_min  radius_max  radius_mean  radius_median  height_min  height_max  height_median  height_mean     volume       area  area_cartesian_bbox  smallest_bounding_width  smallest_bounding_length  smallest_bounding_area  aspect_ratio threshold  max_feret  min_feret  Contour Lengths  Circular  End to End Distance  Image Name
 Molecule Number                                                                                                                                                                                                                                                                                                                                                                
-0                312.321826  135.710289    2.047651    6.802569     4.468766       4.524109   -0.812277    -0.43474      -0.573088    -0.578251 -31.751035  54.908751           106.156919                 5.459192                 13.041404               71.195532      2.388889     lower  13.079421   5.459192        46.737667     False              5.96906  minicircle
+0                312.332803  135.688334    2.028629    6.781691     4.495498       4.558832   -0.811737   -0.436235      -0.572944    -0.579688 -31.829952  54.908751           106.156919                 5.459192                 13.041404               71.195532      2.388889     lower  13.079421   5.459192        47.836563     False              5.96906  minicircle
```

Comment:
Do we know that these changes are because of a scar in `minicricles.spm`?

Reply:
I believe they are. I have added a plot to show the marked scars mask, and there are a few scars for `minicircle.spm` that are inconsequential with regards to the grains that are detected, however will affect quantities such as average image value and the flattening by a **tiny** amount, which seems to affect each grain a little. 

![image](https://user-images.githubusercontent.com/86117496/210341504-d5988691-31ba-4aa3-a53f-e305a2d97a90.png)

I think the spatial and geometric quantities such as the centres and the radii change because the edge pixels of the grains that are just above / below the threshold are shifted by functions in the second round of flattening, and are moved across the threshold for being detected as a grain. I am currently trying to confirm this with some plots of which pixels in the grains change.